mmcv/runner/hooks/hook.py

```diff
@@ -65,3 +69,24 @@ def is_last_epoch(self, runner):
 
     def is_last_iter(self, runner):
         return runner.iter + 1 == runner._max_iters
+
+    def get_trigger_stages(self):
```

Comment:
Does the name "get_triggered_stages" make more sense?

Reply:
ok